core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -160,9 +152,9 @@ object ConsumerGroupCommand {
     }
 
     protected def describeGroup(group: String) {
-      val configs = parseConfigs(opts)
```

Comment:
0.9.0.0 will actually be the first release to include ConsumerGroupCommand. So, I don't think compatibility is an issue. I agree that we need to remove the --config option.